gc/base/MemoryManager.cpp

```diff
@@ -585,6 +586,13 @@ MM_MemoryManager::decommitMemory(MM_MemoryHandle* handle, void* address, uintptr
 	return memory->decommitMemory(address, size, lowValidAddress, highValidAddress);
 }
 
+bool
+MM_MemoryManager::isMemoryAllocationHintSupported(MM_EnvironmentBase* env)
```

Comment:
This function required for prototype only, no need in final code

Reply:
OK. I can remove it later.